plugins/kuttl.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuttl
+spec:
+  version: "v0.1.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUTTL.
+  homepage: https://kudo.dev/
```

Comment:
Can you link to a page that gives information about `kuttl` itself? This directs to the kudo homepage and it's not obvious how you get to kuttl from here.

Reply:
I'll get the web site up by next week.   An example usage is in this PR https://github.com/kudobuilder/operators/pull/244/files
which is a switch from using `kudo` to run tests to using `kuttl`